service/models.py

```diff
@@ -0,0 +1,90 @@
+import logging
+from flask import Flask
+from flask_mongoengine import MongoEngine
+from flask_mongoengine.wtf import model_form
+
+db = MongoEngine()
+
+class DataValidationError(Exception):
+    """ Used for an data validation errors when deserializing """
+    pass
+# class Product(db.Document):
+#     """
+#     Class that represents a product id
+#     """
+#     product_id = db.IntField(required=True)
+
+class Supplier(db.Document):
+    """
+    Suppliers data schema: https://github.com/nyu-devops-fall19-suppliers/suppliers/issues/21
+    """
+    logger = logging.getLogger('flask.app')
+    app = None
+
+    # Table Schema
+    supplierID = db.IntField(required=True)
```

Comment:
I think MongoDB would automatically generate a unique **_id** field for each collection, like **4f4381f4e779897a2c000009**. Thus I believe we don't need manually assign an integer to each supplier.   

Here is a stackoverflow page on [How to search document by oid in mongoengine](https://stackoverflow.com/questions/9988352/how-to-search-document-by-oid-in-mongoengine)